pkgs/by-name/nu/nurl/package.nix

```diff
@@ -47,7 +47,7 @@ rustPlatform.buildRustPackage rec {
         lib.makeBinPath [
```

Comment:
IMO we should suffix here. The user might have a git version in PATH with more support for things and wonders why nurl fails, eg ssh clones?

Reply:
I agree, but that's out of scope for this PR. Please open a PR with the suggested change and I'll approve it.